packages/discord.js/src/structures/VoiceState.js

```diff
@@ -250,6 +250,15 @@ class VoiceState extends Base {
     return this;
   }
 
+  /**
+   * Fetches this voice state.
+   * @param {boolean} [force=true] Whether to skip the cache check and request the API
+   * @returns {Promise<VoiceState>}
+   */
+  fetch(force = true) {
+    return this.guild.voiceStates.fetch(this.member, { force });
```

Comment:
Yes, it resolves, but I think id would be more suitable
```suggestion
    return this.guild.voiceStates.fetch(this.id, { force });
```